Unit2_Applications/friendship-quiz.js

```diff
@@ -0,0 +1,40 @@
+// questions to be randomly asked on page load
+var questionsList = ["What are your/your friend's favorite animals?", "What do I/you hate the most(foodwise)?", "What is my favorite color?"];
+
+
+//randomly chooses a question to display
+function init() {
+  var randomIndex = Math.floor(Math.random() * questionsList.length);
+  document.getElementById("question").innerHTML = questionsList[randomIndex];
+}
+
+window.onload = init;
+
+var answerList = []; //initialize answerList to be empty
+
+
+//adds user-submitted answer to answerList
+function submitAnswer() {
+  var answer = document.getElementById("answerInput").value;
+  var answer2 = answer.toUpperCase();
```

Comment:
I like your use of .toUpperCase() to make answers not case-sensitive. To make your code more elegant, you could combine these two lines to be var answer = document.getElementById("answerInput").value.toUpperCase();